frontend/src/metabase/services.js

```diff
@@ -364,6 +364,10 @@ export const MetabaseApi = {
     formData: true,
     fetch: true,
   }),
+  tableReplaceCSV: POST("/api/table/:tableId/replace-csv", {
+    formData: true,
+    fetch: true,
+  }),
```

Comment:
Shouldn't all new routes be made in RTKQuery?

Reply:
I'd rather not block on that and do all three endpoints together
https://github.com/metabase/metabase/issues/40625